src/collision.cpp

```diff
@@ -222,74 +222,9 @@ static inline void getNeighborConnectingFace(const v3s16 &p,
 		*neighbors |= v;
 }
 
-collisionMoveResult collisionMoveSimple(Environment *env, IGameDef *gamedef,
-		f32 pos_max_d, const aabb3f &box_0,
-		f32 stepheight, f32 dtime,
-		v3f *pos_f, v3f *speed_f,
-		v3f accel_f, ActiveObject *self,
-		bool collideWithObjects)
+static bool addAreaNodeBoxes(const v3s16 &min, const v3s16 &max, IGameDef *gamedef, Map *map, std::vector<NearbyCollisionInfo> &cinfo)
```

Comment:
1. static functions should use snake_case
https://dev.minetest.net/Code_style_guidelines

2. Suggestion: consider to change `v3s16 &` to `v3s16`. Copying once should be faster than dereferencing often.